source/includes/wp-api-v3/_order-refunds.md

```diff
@@ -53,7 +55,44 @@ The refunds API allows you to create, view, and delete individual refunds.
 | `id`           | integer | Tax rate ID. <i class="label label-info">read-only</i>                         |
 | `total`        | string  | Tax total. <i class="label label-info">read-only</i>                           |
 | `subtotal`     | string  | Tax subtotal. <i class="label label-info">read-only</i>                        |
-| `refund_total` | number  | The amount to refund for this tax. <i class="label label-info">write-only</i>  |
+
+### Order refund - Tax lines properties ###
+
+| Attribute            | Type    | Description                                                                              |
+|----------------------|---------|------------------------------------------------------------------------------------------|
+| `id`                 | integer | Item ID. <i class="label label-info">read-only</i>                                       |
+| `rate_code`          | string  | Tax rate code. <i class="label label-info">read-only</i>                                 |
+| `rate_id`            | integer | Tax rate ID. <i class="label label-info">read-only</i>                                   |
+| `label`              | string  | Tax rate label. <i class="label label-info">read-only</i>                                |
+| `compound`           | boolean | Show if is a compound tax rate. <i class="label label-info">read-only</i>                |
```

Comment:
"show if _it's_ a compound" is better maybe?

Reply:
Yeah, that grammar could definitely be improved. Updated in 849b306